.github/workflows/image-builder.yaml

```diff
@@ -0,0 +1,38 @@
+name: image-builder
+
+on:
+  schedule:
+    - cron: "0 6 */14 * *"
+jobs:
+  image-builder:
+    runs-on: [self-hosted, linux]
+    steps:
+    - name: Checkout image-builder
+      uses: actions/checkout@v3
+      with:
+        repository: 'outscale-vbr/image-builder'
+        path: "add-outscale-image"
+        ref: add-outscale-image
+    - name: Use latest Packer
+      uses: hashicorp-contrib/setup-packer@v1
+      with:
+        packer-version: 1.7.10
+    - name: Launch image-builder
+      run: |
+        sudo pip3 install setuptools-rust
+        sudo pip3 install --upgrade pip
+        export PATH=$HOME/.local/bin:$GITHUB_WORKSPACE/image-builder/images/capi/.local/bin:$PATH
+        make deps-osc
+        groupadd -r packer && useradd -m -s /bin/bash -r -g packer packer
+        chown -R packer:packer ${{ github.workspace }}/add-outscale-image
```

Comment:
Why do you execute packer with a user ?

Reply:
https://www.packer.io/plugins/provisioners/ansible/ansible#become-yes
Il doit pas être root